test/DebugInfo/implicitreturn.swift

```diff
@@ -1,3 +1,5 @@
+// REQUIRES: rdar116482190
```

Comment:
Does this fail only on windows? If yes, we should not skip the other platforms.

Reply:
It only fails on Windows, but we also have a history of not revisiting things that fail on Windows. I want to make sure we actually get it fixed before we call the rebranch complete.

Reply:
Disabling the test entirely is not raising the incentive to fix it, it lowers the test coverage on all platforms and raises the risk that a new regression sneaks in while the test is disabled. If you want to escalate the issue, then escalate the issue.

Reply:
I don't think that disabling it non-macOS targets raises the incentive to fix it, but rather decreases it.  The current prevailing culture is to just disable tests on non-Darwin platforms.  In order to get to the point where we can have parity on platforms would be helped by reducing test coverage as it forces all platforms to be equalized.

Reply:
PR https://github.com/apple/swift/pull/69026 fixes this test everywhere, so I've dropped this part of this patch.
This patch only reverts the removal of the xfail in the driver test now.